README.rst

```diff
@@ -941,6 +941,15 @@ Choose from the list of available rules:
 
   The correct case must be used for standard PHP types in phpdoc.
 
+* **phpdoc_types_order**
```

Comment:
That's on purpose because I want the fixer to be extended in the future to have more advanced sorting options, which is something you seem to agree with:

> later, it could be enhanced to more general rule like int|bool|null -> null|int|bool -> null|bool|int

Extending it may require to deprecate the current option depending on what will be added, but at least we won't have to deprecate the fixer itself in favor of a new one.